gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -54,6 +54,72 @@
           "enum": ["log"],
           "description": "Let the request go through and only output logs"
         }]
+      },
+      "operation": {
+        "type": "object",
+        "$id": "#/definitions/operation",
```

Comment:
Can we define those in the policy manifest spec? And then just refer to those?

Reply:
You mean `manifest-schema.json` ?

Reply:
Yes. It could make the validation harder, but sharing those definitions would be good in my opinion.

Reply:
Yes, it would avoid duplication, but I'm not sure if the tooling we have for validation will work correctly. I'll try it.